code/modules/research/rdconsole.dm

```diff
@@ -167,7 +167,11 @@ Nothing else in the console has ID requirements.
 					var/obj/item/card/id/ID = I.GetID()
 					if(istype(ID))
 						logname = "User: [ID.registered_name]"
-			stored_research.research_logs += "[logname] researched node id [id] with cost [json_encode(price)]."
+			for(var/thing in price)
+				to_chat(world, "DEBUG -- [thing]")
+			var/i = stored_research.research_logs.len
+			stored_research.research_logs += null

```

Comment:
???

Reply:
I'm adding a whole list as a single element of another list. I can't just use `+=`, because that merges the two lists together. I googled, and found a post by Lummox saying this is how to do it. If there's a better way, I'll gladly change it.